src/virtual_secondary/managedsecondary.h

```diff
@@ -1,5 +1,5 @@
-#ifndef UPTANE_MANAGEDSECONDARY_H_
-#define UPTANE_MANAGEDSECONDARY_H_
+#ifndef PRIMARY_MANAGEDSECONDARY_H_
```

Comment:
include-guard was not updated. `VIRTUAL_SECONDARY_MANAGEDSECONDARY_H_`?

Reply:
Actually, I did, PRIMARY_MANAGEDSECONDARY_H_ is the one I intended to apply, the first word denotes the namespace.

Reply:
Ok, my bad that's from our naming conventions and it ends up being a bit confusing here.
It could be prefixed by the directory name but it would get quite big in this case. Or we could keep that.